src/renderer/contexts/BackendContext.tsx

```diff
@@ -100,6 +100,7 @@ export const BackendProvider = memo(
                     try {
                         // eslint-disable-next-line no-await-in-loop
                         await ipcRenderer.invoke('restart-backend');
+                        backend.abort();
```

Comment:
I wonder whether we should abort all current requests before restarting. The timeline of 'restart-backend' is as follows:

```
-> call 'restart-backend'
    - stop python process
    - start new python process
<- 'restart-backend' yields back (await resolves)
    - new python process starts running our code (sanic starts the server)
```

I know this because I ran into this in CLI mode. Just because the backend process is open, doesn't mean it's ready.

So right now, we have a race condition. If the server starts up fast enough, we might not abort the old requests in time.

Reply:
IIRC it didn't work right the other way around, but I can try it

Reply:
works fine